### ISSUE TITLE
[BUG] [Formatter] djlint:off inside a tag's attribute list leaks __DJLINT_UNFORMATTED_BLOCK placeholder and corrupts output (regression in 1.40.1)

### ISSUE THREAD

**Issue by databyte** [djlint/djLint#2181]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: macOS 26.5.1 (arm64)
- Python Version: 3.14.3
- djLint Version: 1.40.2 (also reproduces on 1.40.1)
- template language: django

## Issue

When `{# djlint:off #}` … `{# djlint:on #}` is placed **inside a tag's attribute list** (to protect a multi-line attribute value, e.g. an Alpine.js `x-text` expression), the formatter no longer restores the ignored block. Instead it writes its own **internal placeholder** — `/*__DJLINT_UNFORMATTED_BLOCK_N__*` — directly into the output and mangles the tag, producing invalid HTML and dropping the original attribute entirely.

This is a **regression introduced in 1.40.1** — it's clean in 1.40.0 and earlier. It looks like a side effect of the 1.40.1 change *"Preserve full `djlint:off` blocks before formatter passes"* (which appears to have closed #868 and related `djlint:off` issues).

The corruption is **silent**: the formatter exits 0 and writes the broken file, so `--reformat` in a pre-commit/CI setup will happily commit invalid templates. On a larger template the broken nesting also cascades into spurious `H025` "orphan tag" lint errors.

## How To Reproduce

**Input** (`repro.html`):

```html
<p class="text-gray-500"
  {# djlint:off #}
  x-text="
  items.length > 3
    ? items.slice(0, 3).join(', ') + '...'
    : items.join(', ')
  "{# djlint:on #}></p>
```

**Command:**

```bash
djlint repro.html --reformat --profile django --indent 2
```

**Actual output (1.40.1, 1.40.2)** — corrupted; the `x-text` attribute and the `djlint:off`/`djlint:on` block are gone, replaced by an internal sentinel, and the tag is broken:

```html
<p class="text-gray-500" /*__DJLINT_UNFORMATTED_BLOCK_0__* /></p>
```

**Expected output** — block left untouched, exactly as 1.40.0 produces:

```html
<p class="text-gray-500"
  {# djlint:off #}
  x-text="
  items.length > 3
    ? items.slice(0, 3).join(', ') + '...'
    : items.join(', ')
  "{# djlint:on #}></p>
```

**Version bisect:**

| Version | Result |
|---|---|
| 1.40.0 and earlier | ✅ block preserved (correct) |
| **1.40.1** | ❌ placeholder leaked / output corrupted |
| 1.40.2 | ❌ still corrupted |

```bash
uvx djlint@1.40.0 repro.html --reformat --profile django --indent 2   # clean
uvx djlint@1.40.1 repro.html --reformat --profile django --indent 2   # corrupted
uvx djlint@1.40.2 repro.html --reformat --profile django --indent 2   # corrupted
```

Note: placing `djlint:off` *outside* the tag (between elements) is unaffected — only the in-attribute-list placement regressed.
